src/pymortests/analyticalproblems/function.py

```diff
@@ -70,3 +70,14 @@ def test_pickle_by_evaluation(function):
 def test_invalid_expressions():
     with pytest.raises(TypeError):
         ExpressionFunction('(-1 < x[0]) and (x[0] < 1)', 1)
+
+
+def test_random_bitmap_function():
+    from pymor.analyticalproblems.functions import BitmapFunction
+    f = BitmapFunction.random(shape=(10, 10))
+    for i in range(9):
+        for j in range(9):
+            # all values are between 0 and 1
+            assert 0 <= f([i / 10., j / 10.]) <= 1
+            # all values are different
+            assert (f([i/10., j/10.]) != f([(i+1)/10., (j+1)/10.]))
```

Comment:
Is that actually true? First of all, it might happen that two values are the same and I think this would not be a problem (of course with probability close to 0). Further, this check only compares with some of the neighboring points, so the comment might be misleading.